packages/dcos-ui-update-service/buildinfo.json

```diff
@@ -1,8 +1,8 @@
 {
   "single_source": {
     "kind": "url",
-    "url": "http://downloads.mesosphere.com/dcos-ui-update-service/latest/dcos-ui-update-service-afde1262ebf60aeb3b8431ad00d478d63633ae70",
-    "sha1": "f20e4736b6086c57baafb7febb617809fe3f09a9"
+    "url": "http://downloads.mesosphere.io/dcos-ui-update-service/latest/dcos-ui-update-service-8b688f245fe2ce035bc66162155591824da7b96c",
```

Comment:
this should be `.com` instead of `.io` too.

Reply:
✅